File: CodingPractice-master/leetcode/algorithms/python/LCP_1_GuessNumbers.py
# -*- coding: utf-8 -*-
# @Author: Yan_Daojiang
# @Date:   2019-10-04 13:28:58
# @Last Modified by:   Yan_Daojiang
# @Last Modified time: 2019-10-04 13:29:09
class Solution:
    def game(self, guess: List[int], answer: List[int]) -> int:
        count = 0;
        if guess[0] == answer[0]:
            count = count + 1
        if guess[1] == answer[1]:
            count = count + 1
        if guess[2] == answer[2]:
            count = count + 1
        return count

File: CodingPractice-master/leetcode/algorithms/cpp/LCP_1_GuessNumbers.cpp
/*
* @Author: Yan_Daojiang
* @Date:   2019-10-04 13:28:27
* @Last Modified by:   Yan_Daojiang
* @Last Modified time: 2019-10-04 13:28:40
*/
class Solution {
public:
    int game(vector<int>& guess, vector<int>& answer) {
        int count = 0;
        if (guess[0] == answer[0])
            ++count;
        if (guess[1] == answer[1])
            ++count;
        if (guess[2] == answer[2])
            ++count;
        return count;
        
    }
};